README.md

```diff
@@ -430,7 +430,7 @@ If your Prometheus monitors multiple clusters we require the label you defined f
 For example, if your cluster has the Prometheus label `cluster: "my-cluster-name"`, then run this command:
 
 ```sh
-krr.py simple --prometheus-label cluster -l my-cluster-name
+krr.py simple --prometheus-cluster-key cluster -l my-cluster-name
```

Comment:
@aantn I think this requires a change on the robusta-runner as well
see [here](https://github.com/robusta-dev/robusta/blob/28eed8bad2a7f747d2c55028aa71c9ad5ffe345e/playbooks/robusta_playbooks/krr.py#L156)

Reply:
@arikalon1 it should be fine. I'm not deprecating `--prometheus-label`, just adding another option with a name that makes more sense.

I did deprecate `--prometheus-cluster-label` but the runner doesn't pass that by default.